source/partials/next_release/_release.md.erb

```diff
@@ -3,7 +3,6 @@ The New Dashboard Page which was previewed in 18.2.0 is now the default dashboar
 
 <figure class='shrink-figure-75'>
   <%= image_tag 'releases/18.3.0/new-dashboard.png', alt: 'new dashboard page' %>
-  <caption>New Dashboard Page</caption>
 </figure>
 
 This improves performance for large GoCD instances with thousands of pipelines. It does so by:
```

Comment:
Can we replace this with something to the effect of "After we released the improved GoCD Dashboard (beta) in 18.2, we made several bug fixes and performance improvements based on feedback from our users. In 18.3, we have replaced the older version of the dashboard with this improved dashboard which will be available as the default option to all GoCD users."